apiserver/facades/client/application/base_test.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2025 Canonical Ltd.
```

Comment:
I see that this new file seems to be moved from part of the `package_test.go` - just wondering if there’s a specific reason for separating it like this rather than updating the existing file? Would love to better understand the context here.

Reply:
Hi yao le, im trying to make it consistent with the other packages such as application_offer and resources. Most packages also do not seem to have a base suite. I'm not too sure what is the current practice but I felt it was clearer to me when we isolate the base suite from package_test.